ledger-core/application/testutils/launchnet/cloud_configuration.go

```diff
@@ -147,11 +150,17 @@ func generateNodeConfigs(nodes []nodeInfo, cloudSettings CloudSettings) []config
 		adminAPIPort = cloudSettings.API.AdminPort
 	}
 
-	appConfigs := []configuration.Configuration{}
+	appConfigs := make([]configuration.Configuration, 0, len(nodes))
 	for i := 0; i < len(nodes); i++ {
 		role := getRole(&cloudSettings.Virtual, &cloudSettings.Light, &cloudSettings.Heavy)
 		nodes[i].role = role.String()
 
+		// prepare directory for logs
+
+		if err := os.MkdirAll(launchnetPath("logs", "discoverynodes", strconv.Itoa(i)), os.ModeSticky|os.ModePerm); err != nil {
```

Comment:
Don't we need such option when we don't write anything to FS( including logs  )?

Reply:
Now it is optional